controllers/quickstarts_test.go

```diff
@@ -20,13 +20,13 @@ import (
 	"context"
 	"testing"
 
-	"github.com/ghodss/yaml"
 	consolev1 "github.com/openshift/api/console/v1"
 	"github.com/stretchr/testify/assert"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/yaml"
```

Comment:
so this provides the same semantics as removed pkg w/ no changes?

Reply:
yes, i verified that, having one package in the tests and another in the actual code.